features/api_services_list.feature

```diff
@@ -0,0 +1,198 @@
+Feature: Services list api
+
+  @uses.config.contract_token
+  Scenario Outline: Get services list when unattached/attached
+    Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
+    When I run `pro api u.pro.services.list.v1` with sudo
```

Comment:
Let's also guarantee that the API runs fine when running as `non-root`.
Let's add that to the attached scenario as well